src/wallet/rpcdump.cpp

```diff
@@ -510,3 +534,112 @@ UniValue dumpwallet(const UniValue& params, bool fHelp)
+            "Import several types of addresses (private and public keys, transaction addresses/scripts) with only one rescan\n"
+
+            "Arguments:\n"
+            "1. json request array     (json, required) Data to be imported\n"
+            "  [     (json array of json objects)\n"
+            "    {\n"
+            "      \"type\": \"privkey | pubkey | address\", (string, required) Type of address\n"
+            "      \"value\": \"...\",                       (string, required) Value of the address\n"
+            "      \"timestamp\": 1454686740,                (integer, optional) Timestamp\n"
+            "      \"label\": \"...\"                        (string, optional) Label\n"
+            "      \"p2sh\": true | false                    (bool, optional, default=false) Value is a P2SH\n"
+            "    }\n"
+            "  ,...\n"
+            "  ]\n"
+            "2. rescan                 (boolean, optional, default=true)\n"
+
+            "\nExamples:\n" +
+            HelpExampleCli("importmulti", "'[ { \"type\":\"privkey\", \"value\":\"<my private key>\", \"timestamp\":1455191478 },"
+                                          "{ \"type\":\"pubkey\", \"value\":\"<my public key>\", \"label\":\"example 1\", \"timestamp\":1455191480 } ]' true") +
+            HelpExampleCli("importmulti", "'[{ \"type\":\"pubkey\", \"value\":\"<my public key>\", \"label\":\"example 1\", \"timestamp\":1455191464 } ]' false") + HelpExampleRpc("importmulti", "[ { \"type\":\"privkey\", \"value\":\"<my private key>\" },"
+                                                                                                                                                                                                  "{ \"type\":\"pubkey\", \"value\":\"<my public key>\", \"label\":\"example 1\", \"timestamp\":1455191480 } ], true") +
+            HelpExampleRpc("importmulti", "'[{ \"type\":\"pubkey\", \"value\":\"<my public key>\", \"label\":\"example 1\", \"timestamp\":1455191464 } ]', false") +
+
+            "\nResponse is an array with the same size as the input that has the execution result :\n"
+            "  [ { \"result\": true | false } , ... ]\n");
+
+    if (!EnsureWalletIsAvailable(fHelp))
+        return NullUniValue;
+
+    if (params.size() == 1)
+        RPCTypeCheck(params, boost::assign::list_of(UniValue::VARR));
+    else
+        RPCTypeCheck(params, boost::assign::list_of(UniValue::VARR)(UniValue::VBOOL));
+
+    const UniValue& jRequest = params[0];
+    bool fRescan = params.size() > 1 ? params[1].get_bool() : true;
+
+    UniValue jResponse(UniValue::VARR);
+
+    //Locks the wallet
+    LOCK2(cs_main, pwalletMain->cs_wallet);
+    EnsureWalletIsUnlocked();
+
+    bool fRunScan = false;
+    int64_t nLowestTimestamp = (chainActive.Tip() ? chainActive.Tip()->GetBlockTime() : 0); //Use the timestamp of the TIP as the initial value, if chain empty -> 0
```

Comment:
Deactivating the scan if the tip does not exist it's fine. 
Another issue in removing from `if (fRescan) {` the `&& nLowestTimestamp != 0` condition is if one of the requests does not have the timestamp it will try to compare useless timestamps (assuming that a valid timestamp is > 0).

What about: 

```cxx
    bool fRunScan = false;
    int64_t nLowestTimestamp = 0;

    if (chainActive.Tip())
        nLowestTimestamp = chainActive.Tip()->GetBlockTime();
    else
        fRescan = false;

    BOOST_FOREACH (const UniValue& data, jRequest.getValues()) {
        UniValue result = UniValue(UniValue::VOBJ);
        try {
            if (nLowestTimestamp > 0) {
                int64_t timestamp = data.exists("timestamp") ? data["timestamp"].get_int64() : 0;

                if (timestamp < nLowestTimestamp)
                    nLowestTimestamp = timestamp;
            }
```